libraries/anaconda35/Dockerfile

```diff
@@ -0,0 +1,25 @@
+#  $ docker build . -t continuumio/anaconda3:latest -t continuumio/anaconda3:5.3.0
+#  $ docker run --rm -it continuumio/anaconda3:latest /bin/bash
+#  $ docker push continuumio/anaconda3:latest
+#  $ docker push continuumio/anaconda3:5.3.0
+
+ENV LANG=C.UTF-8 LC_ALL=C.UTF-8
+ENV PATH /opt/conda/bin:$PATH
+
+RUN apt-get update --fix-missing && apt-get install -y wget bzip2 ca-certificates \
+    libglib2.0-0 libxext6 libsm6 libxrender1 \
+    git 
+
+RUN wget --quiet https://repo.anaconda.com/archive/Anaconda3-5.3.0-Linux-x86_64.sh -O ~/anaconda.sh && \
+    /bin/bash ~/anaconda.sh -b -p /opt/conda && \
+    rm ~/anaconda.sh && \
+    ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh && \
```

Comment:
For my understanding, are these three lines intended for interactive shell sessions where a person is exec'ing `/bin/bash` in the docker image?

As far as I can understand, these don't become available in `algorithmia-pipe` cause it doesn't run in a shell, which is why we have to repeat the [`loading and activation`](https://github.com/algorithmiaio/langpacks/pull/96/files#diff-445c51300ed7e5b6f9a8cae1e0a196a8R8). Correct me if I'm wrong, cause I'm trying to make sure I understand.

Reply:
You're right, I refactored these lines into the `algorithmia-build` script and verified functionality. This keeps the sourcing and shell ops away from docker which should make this process much less fragile.